src/export/CodeSystemExporter.ts

```diff
@@ -115,14 +116,40 @@ export class CodeSystemExporter {
     resolveSoftIndexing(successfulRules);
     for (const rule of successfulRules) {
       try {
+        if (rule.isInstance) {
+          const instanceExporter = new InstanceExporter(this.tank, this.pkg, this.fisher);
+          const instance = instanceExporter.fishForFHIR(rule.value as string);
+          if (instance == null) {
+            logger.error(
+              `Cannot find definition for Instance: ${rule.value}. Skipping rule.`,
+              rule.sourceInfo
+            );
+            continue;
+          }
+          rule.value = instance;
+        }
```

Comment:
Ah, right.  I missed the `continue`. Good point. There's limited value in reducing repetition if it introduces toomuch additional complexity.  I guess the function could return a boolean indicating if it succeeded or not. Or it could throw and you could catch -- but that seems a bit much, doesn't it? I think I'd prefer the boolean return over the thrown error. Anyway -- if there is a reasonable approach, great. If not, thanks for considering it!